crowbar_framework/app/models/node_object.rb

```diff
@@ -1138,6 +1138,22 @@ def ssh_cmd(cmd)
     end
   end
 
+  def net_rpc_cmd(cmd)
+    case cmd
+    when :power_cycle
+      unless system("net", "rpc", "shutdown", "-f", "-r", "-I", @node.name ,"-U", "Administrator%#{@node[:provisioner][:windows][:admin_password]}")
```

Comment:
That's correct, but if you kill your admin server and restore it for backup, then you might not have samba installed anymore. Also, this is the server side -- not sure this would automatically install the client side.